src/FSharpPlus/Control/Functor.fs

```diff
@@ -81,6 +81,7 @@ type Map =
     static member Map ((x: StringBuilder       , f        ), _mthd: Map) = new StringBuilder (String.map f (string x))
     static member Map ((x: Set<_>              , f        ), _mthd: Map) = Set.map f x
     static member Map ((_: Set2<'T>            , _: 'T->'U), _mthd: Map) = Set2<'U>()
+    static member Map ((x: Nullable<_>         , f        ), _mthd: Map) = Nullable.map f x
```

Comment:
The compilation issue is due to the fact that it chooses Nullable over other overloads when resolving the correct overload. If you look further down in the file you'll notice that other overloads have the priority set using `Default1` and `Default2`.

Reply:
Wow, thanks! That works on my machine.